### DISCUSSION TITLE
Tips & Tricks link on Paste doc

### DISCUSSION THREAD

**Discussion by PoojMenon** [twilio-labs/paste#1944] [General] [answered]
Hi, 

There's a link to Tips and Tricks on the [getting started](https://paste.twilio.design/getting-started/design#using-components) page that just links to the Paste Figma library. Is that the right link?

**Comment by kate-porter** [comment, marked as answer]
Hey @PoojMenon! Good catch, we moved that Tips and Tricks guide into a different Figma project and neglected to update this link. We'll get a fix in shortly.

In the meantime, if you are looking for the Tips and Tricks, you can find them here: https://www.figma.com/file/WiJFrYa9YpIcFUmT06gTn0/Working-with-Figma?node-id=155%3A126

**Reply by kate-porter** [reply]
I'lll circle back here once the link has been updated on the docs site, too.

**Reply by gloriliale** [reply]
This has been updated!
